### PR TITLE
Create new `chia-consensus` crate and make `chia` a meta-crate

### PR DESCRIPTION
**Pull request by Rigidity**

### CODE REVIEW

File: crates/chia-consensus/fuzz/Cargo.toml [hunk 1]
```diff
@@ -11,11 +11,11 @@ cargo-fuzz = true
...
-clvm-utils = { path = "../crates/clvm-utils" }
-clvm-traits = { path = "../crates/clvm-traits" }
-chia-protocol = { path = "../crates/chia-protocol" }
-chia-traits = { path = "../crates/chia-traits" }
-chia-consensus = { path = "../crates/chia-consensus" }
+clvm-utils = { path = "../../clvm-utils" }
+clvm-traits = { path = "../../clvm-traits" }
+chia-protocol = { path = "../../chia-protocol" }
+chia-traits = { path = "../../chia-traits" }
+chia-consensus = { path = "../../chia-consensus" }
```
Comment (arvidn) [inline crates/chia-consensus/fuzz/Cargo.toml:18]: technically, this could just be `..`

File: .github/workflows/build-riscv64.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ jobs:
             -v ${{ github.workspace }}:/ws --workdir=/ws \
             chianetwork/ubuntu-22.04-risc-builder:latest \
             bash -exc '\
-            cargo test --release
+            cargo test --release -p chia-consensus
```
Comment (arvidn) [inline .github/workflows/build-riscv64.yml:43]: I wonder if it was deliberate to omit the `--all`, and really only run these tests

**Review by arvidn — COMMENTED** [review]
there's a tool under `chia-tools` that generate an initial corpus for the `chia-consensus` fuzzers. I don't think it's placing the files in the right place now. I think it also needs to be updated.

File: .github/workflows/build-riscv64.yml (hunk 2 above)
Comment (Rigidity) [reply .github/workflows/build-riscv64.yml:43]: chia-ssl doesn't compile on riscv

**Review by arvidn — APPROVED** [review]

**Pull request merged**